src/test/java/com/artipie/rpm/hm/StorageHasRepoMdTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.hm;
+
+import com.artipie.asto.Content;
+import com.artipie.asto.Key;
+import com.artipie.asto.Storage;
+import com.artipie.asto.memory.InMemoryStorage;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Assertion;
+import org.llorllale.cactoos.matchers.Mismatches;
+
+/**
+ * Tests for {@link StorageHasRepoMd} matcher.
+ *
+ * @since 0.11
+ */
+public class StorageHasRepoMdTest {
+
+    @Test
+    @Disabled
+    public void matchPositive() throws Exception {
+        final Storage storage = new InMemoryStorage();
+        storage.save(
+            new Key.From("repomd.xml"),
+            new Content.From(
+                Files.readAllBytes(
+                    Paths.get("src/test/resources-binary/repodata/StorageHasRepoMdTest/repomd.xml")
+                )
+            )
+        );
+        storage.save(
+            new Key.From("filelists.xml.gz"),
+            new Content.From(
+                Files.readAllBytes(
+                    Paths.get(
+                        "src/test/resources-binary/repodata/StorageHasRepoMdTest/filelists.xml.gz"
```

Comment:
@paulodamaso why do we heed filelists here?

Reply:
@olenagerasimova Yes, if we have the filelists tag in `repomd.xml` then we will need this